### PR TITLE
Upgrade word-wrap for snyk issue

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-ll**
Issue 277

### CODE REVIEW

File: package.json
Comment (sfc-gh-dprzybysz) [inline package.json:51]: let's use the newest 1.2.6

Comment (sfc-gh-ext-simba-ll) [reply package.json]: updated

**Comment by sfc-gh-ext-simba-ll** [comment]
@sfc-gh-igarish 
I'll merge after you approve it.

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Pull request merged**
